pentaho-gwt-widgets/src/org/pentaho/gwt/widgets/themes/public/themes/sapphire/globalSapphire.css

```diff
@@ -114,15 +114,13 @@ input[type='text'],
 textarea,
 .bootstrap .pentaho-dialog input[type='text'] {
   border: 1px solid #CCC;
-  padding: 10px 10px;
-  font-size: 14px;
+  padding: 10px 2px 8px 4px;
   line-height: 17px;
   border-radius: 4px;
-  font-family: opensansregular;

```

Comment:
I think it will change, in order to make font of this field equal to the font of "Page" and "of {allPagesCount}", so this line will be consistent in terms of font/fontsize:
"Page {currentPage} of {allPagesCount}"
As it's done in Onyx and Crystal.